src/libraries/System.Private.CoreLib/src/System/Globalization/NumberFormatInfo.cs

```diff
@@ -155,10 +161,23 @@ private static void VerifyDigitSubstitution(DigitShapes digitSub, string propert
         }
 
         internal bool HasInvariantNumberSigns => _hasInvariantNumberSigns;
+        internal bool AllowHyphenDuringParsing => _allowHyphenDuringParsing;
 
-        private void UpdateHasInvariantNumberSigns()
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Is this AI annotation necessary? This method should only be called on construction or when the property is set, both of which should be very rare.

Reply:
I'll remove the annotation.